src/app/authentication/components/form-login/form-login.component.html

```diff
@@ -0,0 +1,11 @@
+<form class="form-login">
+  <div class="form-group">
+    <label for="inputEmail" class="form-login__label">Email</label>
```

Comment:
@carryall Should we add more style to the label to have cursor hand on hover?

Reply:
✅  Fixed on [ba3c97e](https://github.com/carryall/survey-challenge/pull/23/commits/ba3c97e11ddef9a1c053d77952c80b77e50dd8e1)
![task1-login-label-hover](https://user-images.githubusercontent.com/1772999/100181791-aa1c6500-2f0d-11eb-9fc7-701bf98797e9.gif)